integration-cli/docker_utils.go

```diff
@@ -1046,11 +1046,12 @@ func getContainerState(c *check.C, id string) (int, bool, error) {
 	return exitStatus, running, nil
 }
 
-func buildImageCmd(name, dockerfile string, useCache bool) *exec.Cmd {
+func buildImageCmd(name, dockerfile string, useCache bool, buildArgs ...string) *exec.Cmd {
 	args := []string{"-D", "build", "-t", name}
 	if !useCache {
 		args = append(args, "--no-cache")
 	}
+	args = append(args, buildArgs...)
```

Comment:
Since we have an explicit "buildArgs" parameter, I would think it would be better to let this func add the "--build-arg" text to the command, and then just let the caller pass in just the "foo=bar" part w/o the "--build-arg" part. Just to make the users of this func have to do less work.